modules/exploits/unix/http/splunk_xslt_authenticated_rce.rb

```diff
@@ -0,0 +1,352 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
```

Comment:
Can likely be moved up to great or excellent once verision check is implemented.
https://docs.metasploit.com/docs/using-metasploit/intermediate/exploit-ranking.html